code/modules/power/cell.dm

```diff
@@ -10,6 +10,8 @@
 	throw_speed = 2
 	throw_range = 5
 	w_class = 2
+	var/part_type = "cell" //I blame vehicles
```

Comment:
compareParts actually needs this, otherwise i had some issues with determining whether there is a cell already installed in the vehicle